### DISCUSSION TITLE
Proposal: Add an option to move the "Text Selection Tool" & "Hand Tool" items next to "Save" item

### DISCUSSION THREAD

**Discussion by Aleksandra0056** [black7375/Firefox-UI-Fix#834] [Ideas]
Hello, I have a proposal in PDF reader. I think, that the "Text Selection Tool" & "Hand Tool" items should be next to the save icon/item because they are important items, and I think, that Mozilla made an issue when they moved them into the PDF tools menu instead.

My solution is to add an option to about:config to move these items next to the save item.

I have also another reason why you should add an option to move them next to the save icon. For example in Paint.net or Photoshop app, there are also similar tools right in the toolbar instead of being somewhere in the menu.

#### The red arrows point to mentioned items, and the blue line is where these items should be:
![image](https://github.com/black7375/Firefox-UI-Fix/assets/69351811/f3c8e330-9f12-4beb-bc84-aeb77366c94b)

If it would be possible with pure CSS, then I would be very thankful if you would add an option to move these items next to save items. And it would be better, and faster to change the tool right from the toolbar instead of the menu.

UPDATE 1 - Related Bugzilla issue: https://bugzilla.mozilla.org/show_bug.cgi?id=1867960

**Comment by Aleksandra0056** [comment]
And, I also think, that this change would be useful for touch devices.

**Comment by black7375** [comment]
I think it's difficult to change with just CSS.
How about making a proposal to project `pdf.js`?
- https://github.com/mozilla/pdf.js

**Reply by Aleksandra0056** [reply]
Yes, I did it: https://github.com/mozilla/pdf.js/issues/17366

Can we close this?

**Reply by black7375** [reply]
Yes. we can close this issue.
